chall06/sadawi.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import sys, fileinput, re
+
+def get_biggest(shelfs):
+		biggest = 0
+		for i in range(0, len(shelfs) - 1):
+			if int(shelfs[i]) > int(shelfs[biggest]):
+				biggest = i
+		size = shelfs[biggest]
+		del shelfs[biggest]
+		return int(size)
+
+if len(sys.argv) < 2:
+	lines = []
+	shelfs = input().split()
+	for n in shelfs:
+		if not n.isdigit():
+			sys.exit(sys.argv[0] + ": " + "stdin:" + " Can't read file")
```

Comment:
you should take a look how to format text + variable in python to avoid concatenation like that.

```python
sys.exit(f"{sys.argv[0]}: stdin: Can't read file")
```